### PR TITLE
[improvement][agg]Process aggregated results in the vectorized way

### PR DESCRIPTION
**Pull request by mrhhsg**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Aggregated results are currently processed row by row, this is an inefficient way of accessing memory and will call virtual functions too many times.

Here test on clickbench(https://github.com/ClickHouse/ClickBench)  with SQL:
```sql
SELECT UserID, COUNT(*) FROM hits GROUP BY UserID ORDER BY COUNT(*) DESC LIMIT 10;
```
master:
image

agg_result_vec:
image

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by wangbo — DISMISSED** [review]
LGTM

File: be/src/vec/columns/column_nullable.cpp [hunk 1]
```diff
@@ -152,6 +166,17 @@ void ColumnNullable::serialize_vec(std::vector<StringRef>& keys, size_t num_rows
     get_nested_column().serialize_vec_with_null_map(keys, num_rows, arr.data(), max_row_byte_size);
 }
 
+void ColumnNullable::deserialize_vec(std::vector<StringRef>& keys, const size_t num_rows) {
+    const auto& arr = get_null_map_data();
+    for (size_t i = 0; i != num_rows; ++i) {
```
Comment (HappenLee) [inline be/src/vec/columns/column_nullable.cpp:171]: this code seems can SIMD.

**Comment by adonis0147** [comment]
`Clang build` failed. Please fix it.

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**